src/function/table/table_scan.cpp

```diff
@@ -233,8 +233,17 @@ static void IndexScanFunction(ClientContext &context, TableFunctionInput &data_p
 	auto &transaction = DuckTransaction::Get(context, bind_data.table.catalog);
 	auto &local_storage = LocalStorage::Get(transaction);
 
+	vector<column_t> physical_ids;
```

Comment:
Can we move this to `IndexScanInitGlobal` to avoid doing it for every invocation of `IndexScanFunction`? With this change we perhaps don't need to store `column_ids` at all in `IndexScanGlobalState`?

Reply:
Done, I've also changed some places from column_t to storage_t, it's just semantics but I prefer making it somewhat more obvious these are working with (and expecting) physical indices